test/parallel_api/dynamic_selection/sycl/test_dynamic_load_policy_sycl.pass.cpp

```diff
@@ -7,10 +7,11 @@
 //
 //===----------------------------------------------------------------------===//
 
-#include <iostream>
 #include "oneapi/dpl/dynamic_selection"
+#include <iostream>
```

Comment:
I believe this include and `std::cout` output is not required

Reply:
In this case, it is informational for devices that were skipped. I haven't seen an informational macro in oneDPL so can maintain it for now.